Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/10.3/InspectorBackendCommands.js

```diff
@@ -25,8 +25,7 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-// DO NOT EDIT THIS FILE. It is automatically generated from Inspector-iOS-10.3.json
-// by the script: Source/JavaScriptCore/inspector/scripts/generate-inspector-protocol-bindings.py
+// DO NOT EDIT THIS FILE. It is automatically generated from Inspector-iOS-10.3.json by the script: Source/JavaScriptCore/inspector/scripts/generate-inspector-protocol-bindings.py
```

Comment:
When did this change? 🤔

Reply:
honestly no idea 😅 i wondered the same thing